reader.lua

```diff
@@ -93,6 +93,24 @@ local lfs = require("libs/libkoreader-lfs")
 local UIManager = require("ui/uimanager")
 local Device = require("device")
 local Font = require("ui/font")
+local ConfirmBox = require("ui/widget/confirmbox")
+
+local function retryLastFile()
+    return ConfirmBox:new{
+        text = _("Cannot open last file.\nDo you want to retry?"),
```

Comment:
Shouldn't this offer a little more explanation? I only know what this means (i.e., external storage might not yet be mounted) because I've read the issue. Here's a quick suggestion but don't take it as gospel. ;-)

`Cannot open last file. This could be because it was deleted or because external storage is still being mounted.\nDo you want to retry?`